cn_indian_payroll/cn_indian_payroll/overrides/payroll_configuration.py

```
@@ -0,0 +1,119 @@
+                    
+
+
+
+
+                custom_field_doc.save()
+                frappe.db.commit()  
+
+
+
+
+def salary_component(self):
```

Ruff (0.8.2)

Missing return type annotation for public function `salary_component`

Add return type annotation: `None`

(ANN201)